### ISSUE TITLE
WindowsError

### ISSUE THREAD

**Issue by celorodovalho** [atoha/dbootstrap#65]
I got the following error on Windows 7 64

```
D:\dbootstrap-master>python build.py demo
INFO:dbootstrap.build:Building demo to D:\dbootstrap-master\build\demo
INFO:dbootstrap.build:Building CSS.
WARNING:dbootstrap.build.execute:Command stylus.cmd --include D:\dbootstrap-mast
er\source\nib\lib --include D:\dbootstrap-master\source\dbootstrap\theme\dbootst
rap D:\dbootstrap-master\source\dbootstrap\theme\dbootstrap\index.styl exited wi
th code 8
Traceback (most recent call last):
  File "build.py", line 306, in <module>
    raise SystemExit(main())
  File "build.py", line 77, in main
    dbootstrap_css_path
WindowsError: [Error 2] O sistema nÒo pode encontrar o arquivo especificado
```

**Comment by martinpengellyphillips** [comment]
Hey! Can you confirm whether you can run 'stylus.cmd' from a terminal?

**Comment by celorodovalho** [comment]
Sorry! I have downloaded all the dependencies and put in the folder of dbootstrap, now I can generate a demo.

I think it is best to describe in more detail in the documentation which dependencies are and where to stay.
Because in my case, I downloaded the link (https://github.com/thesociable/dbootstrap/archive/master.zip) and follow the description and it did not work, I believe I was using the git clone, the dependencies would be automatic, most unfortunately do not have that option here.

The topic can be closed!
Thanks! Good work the dbootstrap!
